local_ads/local_ads_tests/campaign_serialization_test.cpp

```diff
@@ -29,7 +29,30 @@ std::vector<Campaign> GenerateRandomCampaigns(size_t number)
     auto const fid = featureIds(gen);
     auto const iconid = icons(gen);
     auto const days = expirationDays(gen);
-    cs.emplace_back(fid, iconid, days, true /* priorityBit */);
+    cs.emplace_back(fid, iconid, days);
+  }
+  return cs;
+}
+
+std::vector<Campaign> GenerateCampaignsV2(size_t number)
+{
+  std::random_device rd;
+  std::mt19937 gen(rd());
```

Comment:
int kSeed = 42;
std::mt19937 gen(kSeed);

Otherwise tests too random, in case of failure it would not be possible to reproduce it.